admin/class-meta-box-content-permissions.php

```diff
@@ -204,7 +204,7 @@ public function update( $post_id, $post = '' ) {
 		$old_message = members_get_post_access_message( $post_id );
 
 		// Get the new message.
-		$new_message = isset( $_POST['members_access_error'] ) ? stripslashes( wp_filter_post_kses( addslashes( $_POST['members_access_error'] ) ) ) : '';
+		$new_message = isset( $_POST['members_access_error'] ) ? stripslashes( wp_filter_post_kses( addslashes( wp_unslash( $_POST['members_access_error'] ) ) ) ) : '';

```

Comment:
The strip/add slashes here is just undoing what `wp_filter_post_kses` does internally, but it can use `wp_kses_post` to avoid that and remove the calls here. The `wp_unslash` call would still be needed for `$_POST`, so yes, the line that you have there would work.